knowledge-base/window-focus-button-textbox-on-open.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Focus Button or TextBox on Window Open
+description: Learn how to focus a button, input, textbox, or any component when the Telerik Window for Blazor opens.
+type: how-to
+page_title: How to Focus Button or TextBox on Window Open
+slug: window-kb-focus-button-textbox-on-open
+position: 
+tags: telerik, blazor, window, dialog
+ticketid: 1486212, 1513605, 1610413, 1659021
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Dialog for Blazor, <br /> Window for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+This KB article answers the following questions:
+
+* How to set default focus to a TextBox or a Button in a TelerikWindow?
```

Comment:
Telerik Window isn't specific enough. Choose one of the following: 
* Blazor Window
* Telerik UI for Blazor Window
* UI for Blazor Window